tsconfig.json

```diff
@@ -7,7 +7,8 @@
         "sourceMap": true,
         "rootDir": "src",
         "noImplicitAny": true,
-        "removeComments": false
+        "removeComments": false,
+        "noFallthroughCasesInSwitch": true

```

Comment:
They were an error already with tslint, but now we are changing to the typescript implementation because it's newer and better.